RELEASE.md

```diff
@@ -1,24 +1,21 @@
 # Release Documentation
 
-## Automatic releases
+## Canary releases
 
-We use [`auto`][auto] to automate releases to each package.
+We use [`auto`][] to automate "canary" releases to each package.
+A "canary" release is a release on `npm` that can be used just by creating a PR.
 
-### PR workflow
-
-When a PR is opened, the correct type of release should be selected: major, minor, or patch.
-After the PR is merged, [`auto`][auto] will create the appropriate release, publish it to `npm`, and report back in the PR what the newly released version is.
-
-### Canary workflow
-
-When a PR is opened, [`auto`][auto] will create a "canary" release, publish it to `npm`, and report back in the PR how to use the "canary" release.
+When a PR is opened, [`auto`][] will create a "canary" release, publish it to `npm`, and report back in the PR how to use the "canary" release.
 The "canary" release can be used to test out some changes without fully merging the PR.
 
-## Manual releases
+## Package releases
+
+We use [`changesets`][] to help with normal package releases.
 
-There is not currently a (documented) manual release process; neither through GitHub Actions nor outside of GitHub Actions.
-If we find that we need/want the ability to manually release through GitHub Actions, we can look into using the [`workflow_dispatch` event][workflow_dispatch] to [manually run a workflow][]
+[`changesets`][] will manage a long-running PR that batches changes together.
+When a normal PR is merged, [`changsets`][] will keep track of any [changeset][]s in this long-running PR.
+When we're ready to release some number of packages, we can merge this long-running PR and [`changesets`][] will release all packages with a [changeset][].
```

Comment:
I didn't quite realize that `changesets` manages a long-running PR to control versioning — this being the case, should we change the repository settings to **prevent squash merges**? I imagine that squashing and merging would cause issues in the long-running branch? Maybe I'm misunderstanding

Reply:
Far as I understand it, it'll keep the long-running branch on top of the base branch whenever it changes, so it shouldn't matter how we merge into the repo. For example, the `changeset-release/main` branch in https://github.com/changesets/changesets/pull/709 was "rebased<sup>[1]</sup>" at least seven times while more changes were merged into their `main` branch. I think `changesets` uses squash merges (at least that's what it looks like) and it doesn't seem to cause any issues.

Long story short, I don't think it matters. But we can change it if we'd prefer it one way or the other.

<sup>[1]</sup>: I don't actually know if it's rebasing or making a whole new branch locally and force pushing it. It really doesn't matter, the outcome is the same.

Reply:
Sounds good! You know I like to Squash & Merge 😂 😭 . If that's the case, we can just leave the settings as-is. If it becomes a problem, it's easy to adjust.